requirements.txt

```diff
@@ -32,7 +33,10 @@ sphinx_rtd_theme==0.4.3
 sphinx-autodoc-typehints==1.10.3
 Sphinx==2.2.0
 subword-nmt==0.3.7
+tensorboard==2.3.0
+tensorboardX==2.1
 tokenizers>=0.8.0
+torchtext==0.7.0
```

Comment:
ugh i'm not sure if this messes up my pytorch 1.4 tests but let's go

Reply:
Changed to >=0.5.0